perma_web/perma/models.py

```diff
@@ -1273,7 +1273,7 @@ class CaptureJob(models.Model):
     human = models.BooleanField(default=False)
     order = models.FloatField(db_index=True)
     submitted_url = models.CharField(max_length=2100, blank=True, null=False)
-    created_by = models.ForeignKey(LinkUser, blank=False, null=False, related_name='capture_jobs')
+    created_by = models.ForeignKey(LinkUser, blank=False, null=False)
```

Comment:
Why are you dropping the related name here? It seems better to have `LinkUser.capture_jobs` than `LinkUser.capture_job_set`.

Reply:
For no good reason. Fixing.

Reply:
(I saw "related_name='capture_jobs'" below for link_batch and was momentarily confused, thought this was a typo)